native/python_package/python_driver/cli.py

```diff
@@ -41,16 +41,7 @@ def get_processor_instance(format_: str, custom_inbuffer: InBuffer=None,
 
 
 def main() -> None:
-    """
-    If you pass the --json command line parameter the replies will be
-    printed using pprint without wrapping them in the msgpack format which
-    can be handy when debugging.
-    """
-
-    if len(sys.argv) > 1 and sys.argv[1] == '--msgpack':
-        format_ = 'msgpack'
-    else:
-        format_ = 'json'
+    format_ = 'json'
 
     processor, inbuffer = get_processor_instance(format_)
```

Comment:
@smola requested some time ago that the json processor would be injected as a dependency trough I'm personally more of the YAGNI approach to these things but for the moment I would prefer to leave it that way.